pkg/rtc/transport.go

```diff
@@ -1412,12 +1390,8 @@ func (t *PCTransport) processEvents() {
 					onNegotiationFailed()
 				}
 			}
-			break
```

Comment:
There is a break from loop in old code on failure. It is not there in the new OpsQueue based change here. But, it is okay as negotiation failure will cause a full reconnect.